src/screens/Profile/index.tsx

```
@@ -0,0 +1,11 @@
+import {useEffect} from 'react';
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)